src/v2/Components/v2/ArtworkFilter/ArtworkFilters/ResultsFilter.tsx

```diff
@@ -0,0 +1,74 @@
+import { Flex, Toggle } from "@artsy/palette"
+import { intersection, sortBy } from "lodash"
+import React from "react"
+import {
+  MultiSelectArtworkFilters,
+  Slice,
+  useArtworkFilterContext,
+} from "../ArtworkFilterContext"
+import { INITIAL_ITEMS_TO_SHOW, ShowMore } from "./ShowMore"
+import { FacetFilter, useFacetFilter } from "./FacetFilter"
+import { ResultOption } from "./ResultOption"
+
+interface ResultsFilterProps {
+  facetName: keyof MultiSelectArtworkFilters
+  slice: Slice
+  label: string
+  placeholder: string
+  expanded?: boolean
+}
+
+export const ResultsFilter: React.FC<ResultsFilterProps> = ({
+  facetName,
+  slice,
+  label,
+  placeholder,
+  expanded = true,
+}) => {
+  const { aggregations, currentlySelectedFilters } = useArtworkFilterContext()
+  const { isFiltered, handleFilterChange } = useFacetFilter()
+
+  const selectedValues = currentlySelectedFilters()[facetName]
+  const results = aggregations.find(aggregation => aggregation.slice === slice)
+    ?.counts
+
+  if (!results?.length) {
```

Comment:
Handles most of https://github.com/artsy/force/pull/7270 (sans time period filter)